package.json

```diff
@@ -11,8 +11,8 @@
     "compile": "tsc && npm run bundle-js && npm run vendor-js",
     "bundle-js": "browserify -e src/index.js -o www/bundle.js --no-bundle-external --debug",
     "vendor-js": "browserify -o www/vendor.js -r fh-js-sdk -r fastclick -r react-dom -r react",
-    "ios-sim": "npm run compile && cordova platform add ios ; && cordova run ios --emulator",
```

Comment:
D'oh, my bad! One thing to note is that the `;` was deliberate, I just forgot to remove the `&&` on the `ios-sim` line. 

The reason for this is that if the platform is already added Cordova returns an error exit code on `add`. The `;` bypasses this since we just wanted to ensure the platform existed. Check this output:

```
> ts-react-hello-world@0.1.0 vendor-js workspaces/ts-react-hello-world
> browserify -o www/vendor.js -r fh-js-sdk -r fastclick -r react-dom -r react

Error: Platform ios already added.
Building project: workspaces/ts-react-hello-world/platforms/ios/RHMAP Hello World using TypeScript and React.xcworkspace
	Configuration: Debug
	Platform: emulator
```

If we use `&&` after the `cordova platform add ios` it works fine initially, but any future calls will fail with `Error: Platform ios already added.`.

What are your thoughts on a potnetial alternative for improving this?